service/crawlingTitleService.js

```diff
@@ -2,22 +2,17 @@ const puppeteer = require("puppeteer");
 
 const getCrawlingTitle = async (req, res) => {
   const decodedLink = decodeURIComponent(req.params.url);
-
-  console.log("decodedLink: ", decodedLink);
-
-  const browser = await puppeteer.launch();
+  const browser = await puppeteer.launch({ headless: true });
 
   try {
     const page = await browser.newPage();
 
     await page.goto(decodedLink);
-
     await page.waitForFunction(
       "window.performance.timing.loadEventEnd - window.performance.timing.navigationStart >= 500"
     );
 
     const element = await page.$("title");
```

Comment:
```suggestion
    const element = await page.$("title");
```

page.$ 는 title태그인 첫 번째 요소를 찾는 것 같습니다.
만약 첫 번째 요소로 의도하지 않은 값이 들어올 때의 에러케이스를 설정하는 것도 좋을 것 같습니다.
혹은 선택자를 확실하게 재조정하는 것도 좋을 것 같습니다.

Reply:
좋은 의견 감사합니다! 
HTML 문서 당 title 태그 값은 하나만 있어야 한다는 정보를 토대로, css Selector와 일치하는 단일 요소를 반환하는 puppeteer의 page.$() 메서드를  사용하였습니다. 
감사합니다!